melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/fra_sed/melosys_eessi_melding/MelosysEessiMeldingMapper.java

```diff
@@ -4,17 +4,21 @@
 import java.util.stream.Collectors;
 
 import no.nav.melosys.eessi.kafka.producers.model.Arbeidssted;
+import no.nav.melosys.eessi.kafka.producers.model.Avsender;
 import no.nav.melosys.eessi.kafka.producers.model.MelosysEessiMelding;
 import no.nav.melosys.eessi.kafka.producers.model.Statsborgerskap;
 import no.nav.melosys.eessi.models.sed.SED;
+import no.nav.melosys.eessi.service.sed.helpers.LandkodeMapper;
 
 public interface MelosysEessiMeldingMapper {
 
     default MelosysEessiMelding map(String aktoerId, SED sed, String rinaDokumentID, String rinaSaksnummer,
-            String sedType, String bucType, String journalpostID, String dokumentID, String gsakSaksnummer, boolean sedErEndring) {
+                                    String sedType, String bucType, String avsenderID, String landkode,
```

Comment:
Hadde vel holdt å ta med kun `avsenderID` og ikke `landkode` da landkode uansett hentes fra `avsenderID`. Og i tillegg at mapping til nav-iso-standard gjøres her

Reply:
!? Mapping til "NAV-ISO" gjøres allerede her linje 21. `landkode` hentes ikke hver gang fra `avsenderID`. Jeg har gjort det slik for å bruke `organisation.getCountryCode()`

Reply:
Ja, da var jeg litt rask der 😄 Men da er det good :+1: